_inc/client/components/settings-card/index.jsx

```diff
@@ -321,9 +321,19 @@ export const SettingsCard = props => {
 		return null;
 	}
 
+	let moduleClassName = '';
+	if ( props.feature ) {
+		moduleClassName = 'jp-settings-' + props.feature;
```

Comment:
Maybe we could use template strings here as well, to be consistent?

Reply:
Done!